src/test/java/Homework19.java

```diff
@@ -0,0 +1,56 @@
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.devtools.v104.console.model.ConsoleMessage;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.openqa.selenium.WebDriver;
+
+import java.time.Duration;
+
+
+public class Homework19 extends BaseTest {
+
+    @Test(priority=1)
+    public void deletePlayList() throws InterruptedException {
+
+        provideEmail("iq14111991@gmail.com");
+        providePassword();
+        clickSubmitBtn();
+        deletePL();
+    }
+    private void deletePL() throws InterruptedException {
+
+        chosePL();
+        getPlaylistLink();
+        deleteChosenPL();
+        verifyRemovedPL();
+    }
+
+    private void chosePL(){
+
+        WebElement chosePL = driver.findElement(By.xpath("//a[contains(text(), 'PlayListsForDeleting')]"));
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/201462289-b64ab572-0efa-4944-9f59-462cb3230c5b.png)

![image](https://user-images.githubusercontent.com/104761482/201462320-edd68e88-44d1-4e42-b306-179c67f678ab.png)

Try like this:

**Step 1** Choose a playlist

![image](https://user-images.githubusercontent.com/104761482/201462587-62c77441-6b6f-4f00-91f8-3ddd7033cbc8.png)


**Step 2** Enter button "Delete this playlist"

![image](https://user-images.githubusercontent.com/104761482/201462602-ba79c852-22fa-49e8-b2fc-d88861e23a81.png)

**Step 3** Validate that the playlist is deleted

![image](https://user-images.githubusercontent.com/104761482/201462646-64d431ef-562b-4080-bb51-30a43eb0a24e.png)

`Assert.assertEquals(getConfirmationPopupText(), "Exampe");`